raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -149,6 +152,15 @@ public static ConfigDef configDef() {
         return new ConfigDef(CONFIG);
     }
 
+    public static List<Node> quorumVoterStringsToNodes(String quorumVotersString) {
+        return parseVoterConnections(Arrays.stream(quorumVotersString.split(","))
+                .filter(part -> !part.isEmpty())
+                .collect(Collectors.toList())).entrySet().stream()
```

Comment:
I think we can avoid this pattern if we refactor `parseVoterConnections` to use a new method called `parseVoterConnection` that knows how to convert a `String` to a `Node`.

Reply:
Same as above. Will refactor into helper functions on RaftConfig.